storage/backend/ceph/ceph_test.go

```diff
@@ -199,3 +199,21 @@ func (s *cephSuite) TestMounted(c *C) {
 	c.Assert(driver.Unmount(driverOpts), IsNil)
 	c.Assert(driver.Destroy(driverOpts), IsNil)
 }
+
+func (s *cephSuite) TestInternalNames(c *C) {
+	driver := NewDriver()
+	out, err := driver.InternalName("tenant1/test")
```

Comment:
@erikh Is 'tenant1/test/foo' an allowed format? From #75 looks like ceph won't allow creating a volume with `/` so just wondering if we should catch it here or let ceph error out.